v23.1/show-ranges.md

```diff
@@ -25,96 +29,567 @@ To use the `SHOW RANGES` statement, a user must either be a member of the [`admi
 
 Parameter | Description
 ----------|------------
-[`table_name`](sql-grammar.html#table_name) | The name of the table you want range information about.
-[`table_index_name`](sql-grammar.html#table_index_name) | The name of the index you want range information about.
-[`database_name`](sql-grammar.html#database_name) | The name of the database you want range information about.
+[`table_name`](sql-grammar.html#table_name) | The name of the [table](show-tables.html) you want [range](architecture/overview.html#architecture-range) information about.
+[`table_index_name`](sql-grammar.html#table_index_name) | The name of the [index](indexes.html) you want [range](architecture/overview.html#architecture-range) information about.
+[`database_name`](sql-grammar.html#database_name) | The name of the [database](show-databases.html) you want [range](architecture/overview.html#architecture-range) information about.
+[`opt_show_ranges_options`](sql-grammar.html#show_ranges_options) | The [options](#options) used to configure what fields appear in the [response](#response).
+
+## Options
+
+The following [options](sql-grammar.html#show_ranges_options) are available to affect the output. Multiple options can be passed at once, separated by commas.
+
+- `TABLES`:  List [tables](show-tables.html) contained per [range](architecture/overview.html#architecture-range).
+- `INDEXES`: List [indexes](indexes.html) contained per [range](architecture/overview.html#architecture-range).
+- `DETAILS`: Add [range](architecture/overview.html#architecture-range) size, [leaseholder](architecture/glossary.html#leaseholder) and other details.
+- `KEYS`:    Include binary [start and end keys](#start-key).
+- `EXPLAIN`: Show the [SQL query](selection-queries.html) that produces the result.
```

Comment:
I am tempted to suggest to not document EXPLAIN. It refers to crdb_internal usage which we are marking as reserved. EXPLAIN was implemented for troubleshooting inside our development team, not for use by customers.